browser/actions/strategicActions.ts

```diff
@@ -0,0 +1,160 @@
+/* Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+// tslint:disable:no-console
```

Comment:
Shouldn't be blanket disabling these. If you want to use these you should add exclusions for only warn and error. https://eslint.org/docs/rules/no-console#options

Reply:
The link seems to only be for eslint, rather than tslint. I can't seem to find how to disable only warn and error for tslint according to https://palantir.github.io/tslint/usage/rule-flags/. Since I'm only using it for this file, I think it makes the most sense to have this comment in this file. Or else, I can change it in the tslint.json file to only allow error, but that would be for the entire project.